client/my-sites/stats/pages/purchase/index.tsx

```diff
@@ -73,14 +76,10 @@ const StatsPurchasePage = ( {
 		if ( ! siteSlug ) {
 			return;
 		}
-		const trafficPageUrl = `/stats/day/${ siteSlug }`;
-		// Redirect to Calypso Stats if:
-		// - the site is not Jetpack.
-		// TODO: remove this check once we have Stats in Calypso for all sites.
-		if ( ! isSiteJetpackNotAtomic && ! config.isEnabled( 'stats/paid-wpcom-stats' ) ) {
-			page.redirect( trafficPageUrl );
+		if ( isVip ) {
```

Comment:
We'll look at WPCOM at a whole in follow up PRs.

Reply:
Sounds great 👍